syslog-ng-helm-chart/templates/compress-job.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.compressor.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: job-script
+data:
+  entrypoint.sh: |-
+      #!/bin/bash
+      ls -lh /var/log/*-syslog
+      for filename in /var/log/*-syslog; do
+       if [[ $(find "$filename" -mtime +{{ .Values.compressor.retention_days }} -print) ]]; then
+         echo "$filename is older than {{ .Values.compressor.retention_days }} days, Compressing."
+         tar -zcvf "$filename"-$(date +%d-%m-%y).tar.gz $filename
```

Comment:
Usually logfiles are merely compressed via gzip and not encapsulated in a tar envelope. Is there a reason it is better this way when running on kubernetes?